lib/hydranorth/preservation_queue.rb

```diff
@@ -0,0 +1,17 @@
+module Hydranorth::PreservationQueue
+
+  QUEUE_NAME = YAML.load(File.read('config/preservation.yml'))[Rails.env]['queue_name']
+
+  def queue
+    $queue ||= ConnectionPool.new({size: 1, timeout: 5}) { Redis.new }
```

Comment:
Should this be done in an initializer in case we ever want to use redis in other places (i mean if this was HN2 i would push for this, but yeah can go either way for HN)

```ruby
# config/intializersredis.rb

$redis = ConnectionPool.new(size: ENV['RAILS_MAX_THREADS'] || 5, timeout: 5) do
  Redis.connect
end
```